convert2rhel/main.py

```diff
@@ -43,6 +49,16 @@ class ConversionPhase:
     POST_PONR_CHANGES = 4
 
 
+# Track the exit codes for different scenarios during the conversion.
+class ConversionExitCodes:
+    # No errors detected during the conversion
+    CLEAN = 0
+    # Some inhibitors appeared during the pre conversion analysis
+    INHIBITOR = 1
+    # Inhibitors that are part of the conversion process
+    CONVERSION_INHIBITOR = 2
```

Comment:
This I feel we should discuss on Slack or something to come up with something better :thinking: 

```suggestion
# Track the exit codes for different scenarios during the conversion.
class ConversionExitCodes:
    # No errors detected during the conversion
    SUCCESSFUL = 0
    # Some inhibitors appeared during the pre conversion analysis
    ANALYSIS = 1
    # Inhibitors that are part of the conversion process
    CONVERSION = 2
```